src/controllers/import.js

```diff
@@ -234,10 +261,34 @@ function ImportController(context) {
     }
   }
 
+  /**
+   * Get the progress of an import job. Results are broken down into the following:
+   * - complete: URLs that have been successfully imported.
+   * - failed: URLs that have failed to import.
+   * - pending: URLs that are still being processed.
+   * - redirected: URLs that have been redirected.
+   * - running - URLs that are currently being imported.
```

Comment:
So this is interesting.  We could do two things I guess with this. 

1. remove running as it will never be non zero.
2. implement a new feature that will allow the content-scraper to message back / use a lamda to update the status.

Reply:
Is there value in constantly updating this field?

- Currently, it will always be `0` or `1` since we have configured only a single Lambda invocation for the import queues, so effectively sequential processing
- The cost of additional writes to the DB each time a new URL begins processing will add up for larger jobs

Reply:
I agree, I think at this point I should just eliminate the concept of running from the api-service.